### DISCUSSION TITLE
[M-Z] (20 Gennaio 2020) Esercizio 2

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#452] [Esami & Esoneri M-Z]
## Traccia

<img width="924" alt="image" src="https://github.com/user-attachments/assets/c94adcea-46ab-4258-a6f4-9f465c1792a3">

**Comment by rimaout** [comment]
<img width="500" alt="image" src="https://github.com/user-attachments/assets/ae7cd126-71e0-43c5-a805-53ed208926ad">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200120.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200120.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/da158666-c2eb-46a7-8a40-ee94335e9370)
